src/megacmdexecuter.cpp

```diff
@@ -38,6 +38,8 @@
 #include <set>
 
 #include <signal.h>
+#include <mega/localpath.h>
+#include <mega/posix/megafs.h>
 
```

Comment:
Those files are internal code from SDK. Apps shouldn't be using them, but only including the `megaapi.h`